content/applications/marketing/events/event_booths.rst

```diff
@@ -0,0 +1,243 @@
+---------------------
+
+To create a booth category from the :guilabel:`Booth Category` page, click the :guilabel:`New`
+button in the upper-left corner to reveal a blank booth category form.
+
+.. image:: event_booths/booth-category-form.png
+   :align: center
+   :alt: A typical booth category form in the Odoo Events application.
+
+Start by entering a name for the booth category in the top :guilabel:`Booth Category` field. This is
+a **requried** field.
+
+To add a corresponding image to the booth category (e.g. a sample photo of how the booth looks),
+click the :icon:`fa-pencil` :guilabel:`(pencil)` icon that appears when the cursor hovers over the
+camera placeholder in the upper-right corner of the booth category form. When clicked, proceed to
+upload the desired image to the booth category form, if needed.
+
+In the :guilabel:`Booth Details` section, users **must** assign a :guilabel:`Product` to the
+category, and it **must** have *Event Booth* set as the *Product Type* on the product form.
+
+And, regardless of the listed price on the *Event Booth* product chosen, the user can input a custom
+:guilabel:`Price` to be applied for this booth category in the field below.
+
+In the :guilabel:`Sponsorship` section, there is a :guilabel:`Create Sponsor` checkbox option. When
+that checkbox is ticked, when a booth belonging to this booth category is booked, the user is
+created as an official *Sponsor* of the event.
+
+When the :guilabel:`Sponsorship` checkbox is ticked, two additional fields appear beneath it:
+:guilabel:`Sponsor Level` and :guilabel:`Sponsor Type`.
+
+Select a desired level of sponsorship from the :guilabel:`Sponsor Level` drop-down field.
+
+.. tip::
+   To modify any existing :guilabel:`Sponsor Level`, select it from the drop-down field, then click
+   the :icon:`fa-arrow-right` :guilabel:`(right arrow)` that appears at the end of the line. Doing
+   so opens a separate page, wherein the :guilabel:`Sponsor Level` name and :guilabel:`Ribbon Style`
+   can be changed, if necessary.
+
+Users can also create a new :guilabel:`Sponsor Level`, by typing in the name of the new level, and
+clicking :guilabel:`Create and edit...` from the resulting drop-down menu.
+
+Doing so reveals a :guilabel:`Create Sponsor Level` pop-up window.
+
+.. image:: event_booths/create-sponsor-level-popup.png
+   :align: center
+   :alt: The Create Sponsor Level pop-up window that appears in the Odoo Events application.
+
+From this pop-up window, confirm the newly-created :guilabel:`Sponsor Level`, and decide what kind
+of :guilabel:`Ribbon Style` should be applied, if any. The :guilabel:`Ribbon Style` options
+available in that drop-down field are: :guilabel:`No Ribbon`, :guilabel:`Gold`, :guilabel:`Silver`,
+and :guilabel:`Bronze`.
+
+If one is selected, that :guilabel:`Ribbon Style` appears with the sponsor's name on the event
+website.
+
+.. note::
+   Clicking :guilabel:`Create` from the resulting drop-down menu in this instance creates the
```

Comment:
which drop-down menu are we referring to right now? I'm in the Create Sponsor Level one, and I don't see "Create".
![image](https://github.com/odoo/documentation/assets/23611711/758d8c2a-f17d-48fd-b7f3-325b5b799fe5)

Reply:
This admonition was in a poor position in the doc, which caused this totally understandable confusion. My apologies for that. I have since moved it to a far more appropriate position in the doc, and it makes a lot more sense now. Thank you for bringing this to my attention 👍